zebra-network/src/peer_set/candidate_set/tests/vectors.rs

```diff
@@ -115,6 +133,93 @@ fn rejects_all_addresses_if_applying_offset_causes_an_underflow() {
     assert!(validated_peers.next().is_none());
 }
 
+/// Test that calls to [`CandidateSet::update`] are rate limited.
+#[test]
+fn candidate_set_updates_are_rate_limited() {
+    // Run the test for enough time for `update` to actually run three times
+    const INTERVALS_TO_RUN: u32 = 3;
+    // How faster should `update` be called than the expected interval
```

Comment:
Nitpick: clarify purpose of constant
```suggestion
    // How many times should `update` be called in each rate limit interval
```